### PR TITLE
MileStone 9: Validate form

### PR DESCRIPTION
**Pull request by developerwaleed**
MileStone 9: Validate form

>Validate Email Field.
>Will force the user to enter Email in Lower Case.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -140,4 +144,15 @@ document.getElementById('closebtnDSK').addEventListener('click', () => {
...
   deskModal.classList.remove('active');
   Deskoverlay.classList.remove('active');
-});
\ No newline at end of file
+});
+
+form.addEventListener('submit', (event) => {
+  let a = false;
+  if (email.value === email.value.toLowerCase()) {
+    a = true;
```
Comment (youmari) [inline index.js]: - [x] it would be better if you give `a` variable better name than a so everyone ends up file can understand what this variable used for

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @caasperr , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error
✔️ Great title and short summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (developerwaleed) [reply index.js]: Thankyou @youmari i have applied the suggestion!

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @caasperr 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**